google/cloud/documentai_toolbox/wrappers/document.py

```diff
@@ -767,3 +767,33 @@ def export_images(
             index += 1
 
         return output_filenames
+
+    def export_hocr_str(self, title: str) -> str:
+        r"""Exports a string hOCR version of the Document.
+
+        Args:
+            title (str):
+                Required. The title for hocr_page and head.
+
+        Returns:
+            str:
+                A string hOCR version of the Document
+        """
+        f = ""
+        f += '<?xml version="1.0" encoding="UTF-8"?>'
+        f += '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">'
+        f += '<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="unknown" lang="unknown">'
+        f += "<head>"
+        f += f"<title>{title}</title>"
+        f += '<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />'
+        f += '<meta name="ocr-system" content="Document AI OCR" />'
+        f += '<meta name="ocr-langs" content="unknown" />'
+        f += f'<meta name="ocr-number-of-pages" content="{len(self.pages)}" />'
+        f += '<meta name="ocr-capabilities" content="ocr_page ocr_carea ocr_par ocr_line ocrx_word" />'
+        f += "</head>"
+        f += "<body>"
+        for page_to_export in self.pages:
+            f += page_to_export.to_hocr(title=title)
+        f += "</body>"
+        f += "</html>"
+        return f
```

Comment:
I get that you need all of the document information. But is there anything preventing accessing that information from an hocr converter file in the converters directory? It just seems strange to me to have all of this stuff in the wrapper classes when we have other convertors in separate locations.

Reply:
Similar to the other export functions, they have functions inside the document wrapper. I just don't think having a file with one function is worth moving that logic out.

Reply:
I guess it makes some sense. I'll take a look later to see if any of those functions could be refactored out to make the wrappers a little cleaner.

However, I would still prefer to see the hocr template in an xml somewhere instead of hard-coding it in the Python. If it's possible. https://github.com/googleapis/python-documentai-toolbox/pull/123#discussion_r1242316351

Reply:
Fixed [#123 (comment) ](https://github.com/googleapis/python-documentai-toolbox/pull/123#discussion_r1242316351)